src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import javax.script.ScriptContext;
+import javax.sound.midi.SysexMessage;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        printWelcomeMessage();
+        decodingUserCommands();
+    }
+
+    public static void printLine(){
+        System.out.println("----------------------------------"
+                +"-------------------------------------------");
+    }
+    public static void printBye(){
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+    public static void printGreeting(){
+        String greeting ="Hello! I'm Duke\n"
+                + "What can I do for you?\n";
+        System.out.print(greeting);
+        printLine();
+    }
+
+    /**
+     * Reads user input and adds it to the task list accordingly
+     * Based on the input, new task of different types (e.g todo, event, deadline)
+     * will be created accordingly
+     */
+    private static void decodingUserCommands() {
+        String input;
+        int count = 0;
+        ArrayList<String> list = new ArrayList<String>();
+        Task tasks[] = new Task[100];
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+        while(!input.equals("bye")){
+            if(input.equals("list")){
+                if(list.size()==0){
+                    System.out.println("The list is empty");
+                }else {
+                    for (int i = 0; i < list.size(); i++) {
```

Comment:
Try to avoid deep nesting
https://nus-cs2113-ay2021s1.github.io/website/schedule/week4/topics.html#w4-6-code-quality-readability